yarn.lock

```diff
@@ -29815,7 +29812,7 @@ node-pty@beta:
   languageName: node
   linkType: hard
 
-"vm2@npm:^3.9.19, vm2@npm:^3.9.3, vm2@npm:^3.9.8":
+"vm2@npm:^3.9.3, vm2@npm:^3.9.8":
```

Comment:
I suggest we get this into `dev` first and merge to this PR https://github.com/aws-amplify/amplify-cli/pull/12986 .
So that we can remove vm2 entries entirely.